python/cuda_cccl/README.md

```diff
@@ -0,0 +1,11 @@
+# `cuda.cccl`: Experimental CUDA Core Compute Library Python module with CCCL headers
+
+## Documentation
+
+Please visit the documentation here: https://nvidia.github.io/cccl/python.html.
+
+## Local development
+
+```bash
+pip3 install .
+```
```

Comment:
Perhaps it's appropriate to document that this package is currently for internal use only and not meant to be used/installed explicitly.

Reply:
Done: c9a4d9676bb245714321d9c679cad773e67fe88a